libminifi/include/core/state/nodes/SchedulingNodes.h

```diff
@@ -85,7 +85,7 @@ class SchedulingDefaults : public DeviceInformation {
 
     SerializedResponseNode penalizationPeriod;
     penalizationPeriod.name = "penalizationPeriodMillis";
-    penalizationPeriod.value = DEFAULT_PENALIZATION_PERIOD_SECONDS*1000;
+    penalizationPeriod.value = std::chrono::duration_cast<std::chrono::milliseconds>(core::DEFAULT_PENALIZATION_PERIOD).count();
```

Comment:
I'm happy to change this, especially since it's shorter, which is always a good thing, but why is using the constructor better?  They compile to the same thing (at least in this small example): https://godbolt.org/z/nh37j1

Reply:
done in https://github.com/apache/nifi-minifi-cpp/pull/1025/commits/c5a16553add98628476bf642a83ea4f62ac222f5

Reply:
I think of `duration_cast` is mainly meant for lossy conversion while the implicit constructors make it clear that no precision loss occurs.